README.md

```diff
@@ -1,54 +1,55 @@
 # Soroban CLI (soroban-cli)
 
-This repo is home to the [Soroban CLI](https://github.com/stellar/soroban-tools/tree/main/cmd/soroban-cli): The command-line multi-tool for running and deploying Soroban contracts.
+This repo is home to the Soroban CLI, the command-line multi-tool for running and deploying Soroban contracts on the Stellar network.
 
+## Documentation
 
-## Install the CLI
-Install [the latest soroban-cli version](https://github.com/stellar/soroban-cli/releases).
-`cargo install --locked soroban-cli`
+For installation options see below, for usage instructions [see the manual](/docs/soroban-cli-full-docs.md).
```

Comment:
Moved the link to the manual to the top, because folks often come to readmes to find out how to use something as well as install it. We have a great manual, so lets get people to it and if it is below the fold under the install instructions it isn't as readily discoverable.